tests/unit/search2_rpc/test_search2_rpc.py

```diff
@@ -50,7 +50,7 @@ def test_show_indexes_error(services):
             res = json.loads(result)
             assert res['error']
             assert res['error']['code'] == -32003
-            assert res['error']['message'] == 'Server error'
+            assert res['error']['message'] == 'Elasticsearch response error'
```

Comment:
Why did this change? This seems like a bug that should be fixed in the search api rather than adjusting the tests. I'm guessing an exception changed and is no longer being caught correctly?

Reply:
Based on the README documentation, -32003 represents an "Elasticsearch response error." This is also asserted on line 52. Additionally, I traced the code, and if it's correct, it should be raising an Elasticsearch response error.

https://github.com/kbase/search_api2/blob/e17f604909f8b204be4180fae914a2a4b2a2fdaf/src/search2_rpc/service.py#L34

Reply:
I think this is probably related to https://github.com/kbase/search_api2/pull/63#discussion_r2087794411

The code in this PR is a revision behind that in the requirements toml file. I'd guess this is fixed in in the alpha 6 release

Reply:
Where is the Alpha6 release? Alpha5 is the vendored repository that we are currently using. Am i missing something?

Reply:
Search requires alpha 6, not alpha 5: https://github.com/kbase/search_api2/blob/develop/pyproject.toml#L14

As far as where that code lives, none of the branches in the jsonrpcbase repo look promising since they're all behind master. I'd extract it from pypi and update the readme to explain where the code came from

Reply:
Extracted jsonrpc alpha6 from PyPI, and the unit test passed.

Reply:
Did you have to make any changes or is it just strict C&P now?

Reply:
This is a direct copy with no changes. The only difference between alpha5 and alpha6 is the line 'message': RPC_ERRORS.get(code, 'Server error') in the main.py file.